docs/ember-intl/app/templates/docs/helpers/format-list.md

```diff
@@ -1,6 +1,6 @@
 # &#123;&#123;format-list&#125;&#125;
 
-Uses [`Intl.ListFormat`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) to join an array of strings.
+Uses [`Intl.ListFormat#format()`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/format) to join an array of strings.
```

Comment:
I did this kind of update to the specific format method, because I think the method resembles much more the helper.
Also, since we introduced `{{format-date-range}}`, we need a way to differentiate between `format()` and `formatRange()`.
In the future, a similar thing might happen if we introduce `{{format-number-range}}`.

Reply:
I think changing the URL to a more specific page is okay, but let's leave the text that end-developers see to what it is now (something that is easy to read and understand). I'll make this change.